model/postgres/postgres_timeline.rb

```diff
@@ -45,6 +45,17 @@ def need_backup?
     false
   end
 
+  def backups
+    blob_storage_client
+      .list_objects(bucket_name: ubid, folder_path: "basebackups_005/")
```

Comment:
What does `_005` mean?

Reply:
legacy wal-e versioning format. It's just how the keys are in the blob store. (still, I guess?)

Reply:
yep it is still used.